index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>My Review Project</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+  <div class="navbar_section">
+    <div class="container-fluid text-center">
+      <div class="row">
+        <div class="col-12 ess">
+            <h1> Sazzad Essohanam</h1>
+            <h2> The men who saved a billion lives</h2>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="body_section">
+    <div class="container-fluid text-center">
+      <div class="row">
+        <div class="col-12">
+           <div class="image">
+            <img src="img/teamwork.png" alt="">
+            <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger</p>
+           </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="text_section">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-12">
```

Comment:
There is a big article here. You can name the section article. It will fasten your website 🚌